.travis.yml

```diff
@@ -41,7 +42,7 @@ jobs:
     - &STANDARD_TEST_JOB
       stage: Test
       php: 7.3
-      env: DRIVER="xdebug"
+      env: DRIVER="none"
```

Comment:
minor change: I'm not changing the build overall here just declaring the script on the none driver instead of Xdebug